SUTrivBot/Models/Commands.cs

```diff
@@ -2,19 +2,114 @@
 using DSharpPlus.CommandsNext.Attributes;
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Text;
+using System.Threading.Channels;
 using System.Threading.Tasks;
+using DSharpPlus.Entities;
 
 namespace SUTrivBot.Models
 {
     public class Commands
     {
         // TODO: Add appropriate commands. Current list: AskQuestion, GetPointsList, GetTotalQuestionCount, Echo
 
+        private Dictionary<string, GameState> games;
+
+        public Commands()
+        {
+            games = new Dictionary<string, GameState>();
+        }
+        
         [Command("echo")]
         public async Task Echo(CommandContext ctx)
         {
-            await ctx.RespondAsync($"👋 Hi, {ctx.User.Mention}!");
+            await ctx.RespondAsync(ctx.RawArgumentString);
+        }
+        
+        /**
+         * [Not Role Restricted]
+         * Print a list of channel related IDs
+         */
+        [Command("ids")]
+        public async Task Ids(CommandContext ctx)
+        {
+            
+            var result = $"Guild ID (ctx.Guild.Id) {ctx.Guild.Id}{Environment.NewLine}";
+            result += $"Channel ID (ctx.Channel.Id) {ctx.Channel.Id}{Environment.NewLine}";
+            await ctx.RespondAsync(result);
+        }
+        
+        /**
+         * [Not Role Restricted]
+         * Print the list of games
+         */
+        [Command("list")]
+        public async Task List(CommandContext ctx)
+        {
+            var response = "";
+            if (games.Count > 0)
+            {
+                response = games.Aggregate("", (acc, x) =>  $"{acc + x.Value.getGameName() + Environment.NewLine}");
+            }
+            else
+            {
+                response = "No games running";
+            }
+            
+            await ctx.RespondAsync(response);
+        }
+
+        
+        /**
+         * [Not Role Restricted]
+         * Initiate a play session in the current channel.
+         * Does nothing if a play session is already going.
+         * This person is "TriviaMaster" for this game
+         */
+        [Command("play")]
+        public async Task Play(CommandContext ctx)

```

Comment:
Consider using this example for some helper tools in getting this kind of data from a command. 
https://github.com/DSharpPlus/Example-Bots/blob/a4b24b5bd2bbeb79d92d321b3b65afa7db2f4745/DSPlus.Examples.CSharp.Ex02/ExampleCommands.cs#L36

Reply:
Nice! Thank you for the resource, this will definitely be helpful - was there a specific helper method/annotation you were thinking of?

I like this kind of thing 
`public async Task Greet(CommandContext ctx, [Description("The user to say hi to.")] DiscordMember member)`

Reply:
Pretty much exactly that. Using the Dependency Injection to get the items you need rather than retrieving them manually from the CommandContext. Same kind of thing for capturing the full string response from the user.